mne_icalabel/features/tests/test_features.py

```diff
@@ -0,0 +1,21 @@
+from mne.datasets import testing
+from mne.io import read_raw
+from mne.preprocessing import ICA
+from numpy.typing import NDArray
+from mne_icalabel.features import topomap
+
+def _create_test_inst_and_ica():
+    directory = testing.data_path() / "MEG" / "sample"
+    raw = read_raw(directory / "sample_audvis_trunc_raw.fif", preload=False)
+    raw.pick_types(eeg=True)
+    raw.load_data()
+    ica = ICA(n_components=5, method="picard")
+    ica.fit(raw)
+    return ica, raw
```

Comment:
Nit-picking again: this approach you choose is very good if you have a code snippet that is repeated in multiple tests.
For now, there is only one test in this file, so there is no need to define this 'loading' step in a separate function.

Also, let's suppose we have multiple tests, e.g. 5, instead of calling 5 times `_create_test_inst_and_ica`, it would be better to load only once and fit only once an ICA, because this step is time-consuming and will impact the speed at which tests runs. Thus, usually across the MNE tests, the dataset and any needed fit is done only once, at the same indentation level as the imports at the top of the test file.

Example: https://github.com/mne-tools/mne-python/blob/6955f977ac12612f38ab30277dcfff94315a0463/mne/preprocessing/tests/test_ica.py#L36-L56

All those variables are defined in the module (file) scope and are accessible directly within all the functions defined in that same module (file).

Reply:
Okay, I thought we are going to write one script to test all the features. Will rename it.